Correct typo in 24_typical_flash_part_layout.md

grahammc-bit opened the pull request:
Update 24_typical_flash_part_layout.md

Updated the following lines in 24_typical_flash_part_layout.md:
   Line 43: GUIED -> GUIDED
   Line 45: facility -> facilitate

Contributed-under: TianoCore Contribution Agreement 1.1
Signed-off-by: Matthew Graham <matthew.graham@amd.com>

grahammc-bit committed:
Correct typo in 24_typical_flash_part_layout.md

Update facility -> facilitate. I believe this was the intended verbiage from the original author.

makubacki commented:
Commit message requires Signed-off-by and Contributed-under tags

See: https://github.com/tianocore-docs/edk2-TemplateSpecification/wiki/TianoCore-Documents-Commit-Message-Format

makubacki commented on 2_design_discussion/24_typical_flash_part_layout.md:
```diff
@@ -42,7 +42,7 @@ the tools for creating a compressed image and a library for decompressing the
 verification mechanisms are applied to a GUIED encapsulation section. Each
 method needs a unique GUID, however the methods may be applied to images more
-than once per FD image. This is done in order to facility recovery and updates
+than once per FD image. This is done in order to facilitate recovery and updates
```
Commit message requires Signed-off-by and Contributed-under tags

See: https://github.com/tianocore-docs/edk2-TemplateSpecification/wiki/TianoCore-Documents-Commit-Message-Format

grahammc-bit replied:
Thanks, Michael. Added Contributed-under and Signed-off-by fields. Please let me know if this is sufficient.

makubacki reviewed (approved):
Content looks good, but need the commit message updated.

bexcran commented on 2_design_discussion/24_typical_flash_part_layout.md:
```diff
@@ -42,7 +42,7 @@ the tools for creating a compressed image and a library for decompressing the
 image must be provided. These non-standard compression, encryption, signing or
 verification mechanisms are applied to a GUIED encapsulation section. Each
```
Could you fix this at the same time please?
I think it should be GUIDed.

grahammc-bit replied:
I'll make the change but will need confirmation regarding validity. Thank you for the suggestion.

grahammc-bit committed:
Update 24_typical_flash_part_layout.md

Update 24_typical_flash_part_layout.md

Updated the following lines in 24_typical_flash_part_layout.md:
   Line 43: GUIED -> GUIDED
   Line 45: facility -> facilitate

Contributed-under: TianoCore Contribution Agreement 1.1
Signed-off-by: Matthew Graham <matthew.graham@amd.com>

makubacki commented:
Can you please add the `Contributed-under` and `Signed-off-by` tags to the second commit as well?

mdkinney commented:
I see this is targeting a release branch.  This change look like it should go to the master branch first and then a different PR to apply to 1.28 branch if that is required.  A change to master branch would be included in the next official release.
